crates/voicevox_core/src/user_dict/dict.rs

```diff
@@ -0,0 +1,85 @@
+use derive_getters::Getters;
+use fs_err::File;
+use indexmap::IndexMap;
+use itertools::join;
+use std::collections::HashMap;
+use uuid::Uuid;
+
+use super::word::*;
+use crate::{Error, Result};
+
+/// ユーザー辞書。
+#[derive(Clone, Debug, Default, Getters)]
+pub struct UserDict {
+    words: IndexMap<Uuid, UserDictWord>,
+}
```

Comment:
順序が固定されるの、良い仕様だと思いました！　テスト書きやすいし。
まあなくなっても良い仕様かもですが、何も書いてないと流石に意図がわからないと思うのでコメント書いとくのはどうでしょう。
```suggestion
/// ユーザー辞書。
/// 単語はJSONとの相互変換のために挿入された順序を保つ。
#[derive(Clone, Debug, Default, Getters)]
pub struct UserDict {
    words: IndexMap<Uuid, UserDictWord>,
}
```